File: app/src/main/java/com/fitc/wifihotspot/OnStartTetheringCallback.java
package com.fitc.wifihotspot;

public abstract class OnStartTetheringCallback {
        /**
         * Called when tethering has been successfully started.
         */
        public abstract void onTetheringStarted();

        /**
         * Called when starting tethering failed.
         */
        public abstract void onTetheringFailed();

}

File: app/src/main/java/com/fitc/wifihotspot/OnStartTetheringCallback.kt
package com.fitc.wifihotspot

abstract class OnStartTetheringCallback {
    /**
     * Called when tethering has been successfully started.
     */
    abstract fun onTetheringStarted()

    /**
     * Called when starting tethering failed.
     */
    abstract fun onTetheringFailed()
}